terraform/modules/app/task_definition.tf

```diff
@@ -81,6 +82,25 @@ module "task_definition" {
   task_role_arn = var.task_role_arn
 }
 
+resource "aws_ecs_task_definition" "bootstrap" {
```

Comment:
Should we have a `lifecycle { ignore_changes = all }` on this resource?

I don't think it's a huge issue it not being there, but something like this will probably happen:

* First terraform run - task definition created, and service told to use that task definition
* aws cli used to change the service's task definition
* Future terraform runs may update the task definition, but it won't be re-attached to the service because there's a lifecycle rule `ignore_changes = [task_definition]`

If we put an `ignore_changes = all` on this then we'd avoid needlessly updating the bootstrap task definition.

Reply:
https://www.terraform.io/docs/language/meta-arguments/lifecycle.html

> Instead of a list, the special keyword all may be used to instruct Terraform to ignore all attributes, which means that Terraform can create and destroy the remote object but will never propose updates to it.

Reply:
@richardTowers sure, thanks for the suggestion!